tests/spdx3/model/software/test_file.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2023 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from unittest import mock
+
+import pytest
+
+from spdx3.model.software.software_purpose import SoftwarePurpose
+
+from spdx3.model.software.file import File
+
+
+@mock.patch("spdx3.model.creation_information.CreationInformation", autospec=True)
+def test_correct_initialization(creation_information):
+    file = File("SPDXRef-File", creation_information, content_identifier="https://any.uri",
```

Comment:
I'm in favor of introducing a fixture module (analogous to the SPDX2 tests) that includes valid defaults for all class initializations. We could then start to use it here, too.

Reply:
Agreed, opened an issue for that: #432